src/components/protocol_handler/src/protocol_packet.cc

```diff
@@ -47,6 +48,30 @@ namespace protocol_handler {
 
 CREATE_LOGGERPTR_GLOBAL(logger_, "ProtocolHandler")
 
+namespace {
+std::string StringifyFrameType(uint8_t type) {
+  switch (type) {
+    case FRAME_TYPE_SINGLE:
+      return "FRAME_TYPE_SINGLE";
+
+    case FRAME_TYPE_CONSECUTIVE:
+      return "FRAME_TYPE_CONSECUTIVE";
+
+    case FRAME_TYPE_CONTROL:
+      return "FRAME_TYPE_CONTROL";
+
+    case FRAME_TYPE_FIRST:
+      return "FRAME_TYPE_FIRST";
+
+    default:
+      LOG4CXX_ERROR(logger_, "Unknown frame type:" << static_cast<int>(type));
+      return "";
```

Comment:
@KVGrygoriev you can change line 68 ' return "" ' --> 'break'.
And also you can return instead of "" more readable result like "UNRECOGNIZED_FRAME_TYPE" since it used just for logging